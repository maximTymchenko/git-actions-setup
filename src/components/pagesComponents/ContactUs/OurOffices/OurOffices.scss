@import '../../../../styles/vars';

.our-offices {
    &__cities {
        display: grid;
        grid-template-columns: repeat(4, 1fr);

        @media screen and (max-width: $small__desktop) {
            grid-template-columns: repeat(2, 1fr);
        }
        @media screen and (max-width: $mobile) {
            grid-template-columns: repeat(1, 1fr);
        }
    }

    &__city {
    }
    &__timezone {
        display: block;
        margin-bottom: 5px;
        font-family: 'OktaNeue-Black';
        font-size: 11px;
        font-weight: 900;
        color: $text__body;
    }
}
