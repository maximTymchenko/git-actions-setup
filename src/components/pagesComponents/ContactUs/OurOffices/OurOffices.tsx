import React, { FC } from 'react';
import SectionTitle from '../../../common/SectionTitle';
import Clock from '../../../common/Clock';
import './OurOffices.scss';

const OurOffices: FC = () => {
    return (
        <div data-testid="our-offices" className="our-offices__wrapper">
            <SectionTitle type="main" title="Our offices" />
            <div className="our-offices__cities">
                <div className="our-office__city">
                    <SectionTitle className="title" type={'city'} title={'Ukraine HQ'} />
                    <span className="our-offices__timezone">+2 UTC/EET</span>
                    <Clock timezone={'Europe/Kiev'} />
                    <a href="tel:+380931042757" className="contacts__phone">
                        +380–93–104–27–57
                    </a>
                    <a href="mailto:info@blackbird-lab.com" className="contacts__email">
                        info@blackbird-lab.com
                    </a>
                    <span className="contacts__phone">16 Shota Rustaveli St Kyiv,</span>
                    <span className="contacts__phone">02000 Ukraine</span>
                </div>
                <div className="our-office__city">
                    <SectionTitle className="title" type={'city'} title={'Minneapolis'} />
                    <span className="our-offices__timezone">-6 CST</span>
                    <Clock timezone={'America/Belize'} />
                    <a href="tel:+13217541350" className="contacts__phone">
                        +1–321–754–1350
                    </a>
                    <a href="mailto:SF_sales@blackbird-lab.com" className="contacts__email">
                        SF_sales@blackbird-lab.com
                    </a>
                </div>
                <div className="our-office__city">
                    <SectionTitle className="title" type={'city'} title={'Toronto'} />
                    <span className="our-offices__timezone">GMT -5</span>
                    <Clock timezone={'America/Toronto'} />
                    <a href="tel:+353871170708" className="contacts__phone">
                        +353–87–117–0708
                    </a>
                    <a href="mailto:SF_sales@blackbird-lab.com" className="contacts__email">
                        dublin_prod@blackbird-lab.com
                    </a>
                </div>
                <div className="our-office__city">
                    <SectionTitle className="title" type={'city'} title={'Los Angeles'} />
                    <span className="our-offices__timezone">GMT -8</span>
                    <Clock timezone={'America/Los_Angeles'} />
                    <a href="tel:+61391310770" className="contacts__phone">
                        +61–391–31–0770
                    </a>
                    <a href="mailto:SF_sales@blackbird-lab.com" className="contacts__email">
                        AU@blackbird-lab.com
                    </a>
                </div>
            </div>
        </div>
    );
};

export default OurOffices;
