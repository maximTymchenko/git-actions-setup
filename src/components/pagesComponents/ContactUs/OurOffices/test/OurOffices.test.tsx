import React from 'react';
import OurOffices from '../OurOffices';
import { render } from '@testing-library/react';
import renderer from 'react-test-renderer';

it('renders elem correctly', () => {
    const { getByTestId } = render(<OurOffices></OurOffices>);
    expect(getByTestId('our-offices')).toBeTruthy();
});

it('renders without unexpected changes', () => {
    const tree = renderer.create(<OurOffices></OurOffices>).toJSON();
    expect(tree).toMatchSnapshot();
});
