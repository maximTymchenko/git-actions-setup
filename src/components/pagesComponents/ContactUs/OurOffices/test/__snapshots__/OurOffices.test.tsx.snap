// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders without unexpected changes 1`] = `
<div
  className="our-offices__wrapper"
  data-testid="our-offices"
>
  <h2
    className="title title--main"
    data-testid="section-title"
  >
    Our offices
  </h2>
  <div
    className="our-offices__cities"
  >
    <div
      className="our-office__city"
    >
      <h2
        className="title title--city"
        data-testid="section-title"
      >
        Ukraine HQ
      </h2>
      <span
        className="our-offices__timezone"
      >
        +2 UTC/EET
      </span>
      <span
        className="clock"
        data-testid="clock"
      >
        03:01 PM
      </span>
      <a
        className="contacts__phone"
        href="tel:+380931042757"
      >
        +380–93–104–27–57
      </a>
      <a
        className="contacts__email"
        href="mailto:info@blackbird-lab.com"
      >
        info@blackbird-lab.com
      </a>
      <span
        className="contacts__phone"
      >
        16 Shota Rustaveli St Kyiv,
      </span>
      <span
        className="contacts__phone"
      >
        02000 Ukraine
      </span>
    </div>
    <div
      className="our-office__city"
    >
      <h2
        className="title title--city"
        data-testid="section-title"
      >
        Minneapolis
      </h2>
      <span
        className="our-offices__timezone"
      >
        -6 CST
      </span>
      <span
        className="clock"
        data-testid="clock"
      >
        07:01 AM
      </span>
      <a
        className="contacts__phone"
        href="tel:+13217541350"
      >
        +1–321–754–1350
      </a>
      <a
        className="contacts__email"
        href="mailto:SF_sales@blackbird-lab.com"
      >
        SF_sales@blackbird-lab.com
      </a>
    </div>
    <div
      className="our-office__city"
    >
      <h2
        className="title title--city"
        data-testid="section-title"
      >
        Toronto
      </h2>
      <span
        className="our-offices__timezone"
      >
        GMT -5
      </span>
      <span
        className="clock"
        data-testid="clock"
      >
        08:01 AM
      </span>
      <a
        className="contacts__phone"
        href="tel:+353871170708"
      >
        +353–87–117–0708
      </a>
      <a
        className="contacts__email"
        href="mailto:SF_sales@blackbird-lab.com"
      >
        dublin_prod@blackbird-lab.com
      </a>
    </div>
    <div
      className="our-office__city"
    >
      <h2
        className="title title--city"
        data-testid="section-title"
      >
        Los Angeles
      </h2>
      <span
        className="our-offices__timezone"
      >
        GMT -8
      </span>
      <span
        className="clock"
        data-testid="clock"
      >
        05:01 AM
      </span>
      <a
        className="contacts__phone"
        href="tel:+61391310770"
      >
        +61–391–31–0770
      </a>
      <a
        className="contacts__email"
        href="mailto:SF_sales@blackbird-lab.com"
      >
        AU@blackbird-lab.com
      </a>
    </div>
  </div>
</div>
`;
